templates/show-task.html

```diff
@@ -3,16 +3,22 @@
 {% block scripts %}
     <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/bootstrap-table/1.12.1/bootstrap-table.min.css">
     <script src="//cdnjs.cloudflare.com/ajax/libs/bootstrap-table/1.12.1/bootstrap-table.min.js"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/awesomplete/1.1.2/awesomplete.css" />
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/awesomplete/1.1.2/awesomplete.js" async></script>
```

Comment:
We should be loading this stuff from the Toolforge CDN, no? That also applies to the bootstrap includes above.

https://wikitech.wikimedia.org/wiki/Help:Toolforge/Web#External_assets

Reply:
Ah, this page (https://tools.wmflabs.org/cdnjs/) says that these are Wikipedia privacy policy friendly. Hence, I loaded it from there.

Reply:
The idea is that the Toolforge CDN (https://tools.wmflabs.org/cdnjs) is a privacy policy-friendly copy of cdnjs: instead of loading assets from CloudFlare's cdnjs, which sends the user's IP address etc. to CloudFlare, we'd be loading from the Toolforge CDN so the request never goes to a third-party.

So basically we'd be linking to, for example, https://tools-static.wmflabs.org/cdnjs/ajax/libs/awesomplete/1.1.2/awesomplete.js to include awesomplete.js 1.1.2. Same for all of our assets.

Reply:
Got it! I've made the required changes.